cmd/experimental/kjobctl/pkg/cmd/create/create_test.go

```diff
@@ -1006,6 +1010,65 @@ error_path=$(unmask_filename "$SBATCH_ERROR")
 			},
 			wantOutPattern: `job\.batch\/.+ created\\nconfigmap\/.+ created`,
 		},
+		"should create slurm with --priority flag": {
+			beforeTest: beforeSlurmTest,
+			afterTest:  afterSlurmTest,
+			args: func(tc *createCmdTestCase) []string {
+				return []string{
+					"slurm",
+					"--profile", "profile",
+					"--",
+					"--priority", "sample-priority",
+					tc.tempFile,
+				}
+			},
+			kjobctlObjs: []runtime.Object{
+				wrappers.MakeJobTemplate("slurm-job-template", metav1.NamespaceDefault).
+					WithContainer(*wrappers.MakeContainer("c1", "bash:4.4").Obj()).
+					Obj(),
+				wrappers.MakeApplicationProfile("profile", metav1.NamespaceDefault).
+					WithSupportedMode(*wrappers.MakeSupportedMode(v1alpha1.SlurmMode, "slurm-job-template").Obj()).
+					Obj(),
+			},
+			kueueObjs: []runtime.Object{
+				&kueue.WorkloadPriorityClass{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "sample-priority",
+					},
+				},
+			},
+			gvks: []schema.GroupVersionKind{
+				{Group: "batch", Version: "v1", Kind: "Job"},
+				{Group: "", Version: "v1", Kind: "ConfigMap"},
+			},
+			wantLists: []runtime.Object{
+				&batchv1.JobList{
+					TypeMeta: metav1.TypeMeta{Kind: "JobList", APIVersion: "batch/v1"},
+					Items: []batchv1.Job{
+						*wrappers.MakeJob("", metav1.NamespaceDefault).
+							Label(builder.WorkloadPriorityClassLabel, "sample-priority").
```

Comment:
Can we add a method on wrapper like `Profile()`?

Reply:
Done.